packages/core/src/extensions/SideMenu/SideMenuPlugin.ts

```diff
@@ -146,12 +146,12 @@ function setDragImage(view: EditorView, from: number, to = from) {
   dragImageElement.className =
     dragImageElement.className + " bn-drag-preview " + inheritedClasses;
 
-  document.body.appendChild(dragImageElement);
+  view.root.appendChild(dragImageElement);
```

Comment:
Causes an error on drag & drop, need to check if `view.root instanceof ShadowRoot` otherwise append to `view.root.body`